File: problemas/1000/1019.js
let n = require('fs').readFileSync('/dev/stdin', 'utf-8');

const horas = Number(n / 3600);
n %= 3600;
const minutos = Number(n / 60);
n %= 60;
const segundos = n;

console.log(`${horas}:${minutos}:${segundos}`);

File: problemas/1000/1019.dart
import 'dart:io';

void main() {
  int n = int.parse(stdin.readLineSync());

  final horas = n ~/ 3600;
  n %= 3600;
  final minutos = n ~/ 60;
  n %= 60;
  final segundos = n;

  stdout.writeln('$horas:$minutos:$segundos');
}